### DISCUSSION TITLE
Audio

### DISCUSSION THREAD

**Discussion by DCribbs** [LizardByte/community#261] [Sunshine]
Is their anyway to setup Sunshine to capture and stream the selected audio device, without changing the default device on the PC? 

Trying to get it setup, so can launch games on TV, and still use pc audio at same time, like streaming Spotify or something, and only have game audio playing on TV.

Only workaround I have found is, to individually set each apps default audio device to PC speakers, but then I have to go back and change all of that at times when I want to use headset for PC audio.

**Comment by DCribbs** [comment]
Been a while, just checking to see if anyone has any ideas on this?

**Reply by scottpanton** [reply]
I have this issue too. It's not possible currently. I'm using VoiceMeeter to route audio and would like to have Sunshine use the Steam virtual device without altering the default so I can control what is playing back on that device and on physical devices without needing to constantly reconfigure everything. It seems like it would be a very simple change to add a config option to skip altering the default device.
